### ISSUE TITLE
bad-plugin-value error in version 2.14.2 and 2.14.3

### ISSUE THREAD

**Issue by danawillow** [PyCQA/pylint#7005]
### Bug description

When upgrading from 2.14.1 to 2.14.2 or 2.14.3, our pylint stops working with this error: `Command line or configuration file:1:0: E0013: Plugin 'recidiviz.tools.lint.pylint_checkers' is impossible to load, is it installed ? ('No module named 'recidiviz'') (bad-plugin-value)`

Reproduction steps:
1. `git clone https://github.com/Recidiviz/pulse-data.git`
2. `cd pulse-data`
3. `pipenv shell`
4. `pipenv sync --dev`
5. `pip install pylint==2.14.2`
6. `pylint recidiviz/server_config.py`

(there is other pylint output there because this repo is a 2 week old copy of the private repository we're actually doing work in. I can repro the issue in both, but don't get the additional output in the up-to-date one)

### Configuration

```ini
I'm not sure what's causing the bug, so here's a few relevant configs:
https://github.com/Recidiviz/pulse-data/blob/main/.pylintrc
https://github.com/Recidiviz/pulse-data/blob/main/setup.cfg
https://github.com/Recidiviz/pulse-data/blob/main/recidiviz/tools/lint/pylint_checkers.py
```

### Command used

```shell
pylint recidiviz/server_config.py (but that specific file isn't important- it fails on all of them)
```

### Pylint output

```shell
************* Module Command line or configuration file
Command line or configuration file:1:0: E0013: Plugin 'recidiviz.tools.lint.pylint_checkers' is impossible to load, is it installed ? ('No module named 'recidiviz'') (bad-plugin-value)

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```

### Expected behavior

Expect no error

### Pylint version

```shell
pylint 2.14.2
astroid 2.11.6
Python 3.9.11 (main, Apr 29 2022, 09:32:35)
[Clang 13.1.6 (clang-1316.0.21.2)]
```

### OS / Environment

mac OS 12.3.1

### Additional dependencies

(I don't think you'll need this to repro the issue but let me know)

**Comment by jacobtylerwalls** [comment]
Hi @danawillow, thanks for the report.

I see an `init-hook` with quoted code in your config, which suggests to me this is the same issue as #7006. Would you be willing to try unquoting the code and see if that A.) provides a short-term workaround, and B.) confirms this to be the same root cause?
